src/style.css

```diff
@@ -1,3 +1,104 @@
-body {
+html {
+color: #000;
+font-style: normal;
+font-weight: 500;
+line-height: normal;
+text-transform: uppercase;
+  background: gray;
+  font-family: Roboto, sans-serif;
+}
+
+body, ul, li {
   margin: 0;
 }
+
+.container {
+  width: 100vh;
+}
+
+.nav {
+padding-left: 0;
+margin-right: 50px;
+display: flex;
+align-items: center;
+
+}
+
+.nav-link {
+  font-size: 12px;
+  padding: 23px 0;
+  display: inline-block;
+  color: black;
+  text-decoration: none;
+
+}
+
+.nav-item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav-list {
+  padding-inline-start: 0;
+  margin-top: 0;
+  margin-bottom: 0;
+  display: flex;
+  list-style: none;
+  align-items: center;
+
+
+}
+
+.header {
+  justify-content: space-between;
+  display: flex;
+  background-color:white;
+
+}
+
+.img {
+  padding: 10px 50px;
+  display: block;
+  width: 40px;
+  height: 40px;
+}
+
+.is-active::after {
+  background: #00ACDC;
+  width: 100%;
+  display: block;
+  position: absolute;
+  content: "";
+  height: 4px;
+  border-radius: 8px;
+  bottom: 0;
+
+}
+
+.is-active {
+  color: #00ACDC;
+  position: relative;
+}
+
+.is-active:target {
+  color: #00ACDC;
+}
+
+.is-active:active {
+  color: #00ACDC;
+}
+
+:hover[data-qa='hover'] {
+  color: #00ACDC;
+}
+
+:hover {
+  color: #00ACDC;
+}
+
+.nav-item:first-child {
+  margin-left: 0;
+}
+
+.nav-item:last-child {
+  margin-right: 0;
+}
```

Comment:
Redundant, you already used
![image](https://github.com/mate-academy/layout_moyo-header/assets/54936044/5d2d94b0-8c67-43fd-87e9-2c0bf6ecdddf)